### DISCUSSION TITLE
Option for "Label Sounds" to reuse existing label track

### DISCUSSION THREAD

**Discussion by pbienst** [audacity/audacity#4907] [Q&A] [answered]
### Your idea

I'm trying to efficiently export some sentences without leading and trailing silence from an audio file. The problem is that 'Label Sounds' creates a new label track each time you call it on a selection, and 'Export Multiple' only looks at the first label track.

A way to facilitate this workflow would be nice.

### Problem to be solved

See above.

**Comment by SteveDaulton** [comment, marked as answer]
To make Label Sounds add new labels to an existing label track, select both the audio track that you wish to analyze and the label track that you want the labels to go into.
This also applies to any other Nyquist plug-in that creates labels.

**Comment by HighPriest** [comment]
This is a very useful feature, which is currently broken by #5648 

I would like this request to be extended with an option to create label track for each audio track being labeled.
Currently, when we want to batch label multiple tracks at once, a single label track is created at the bottom and all the labels are squished into this single track.
![image](https://github.com/audacity/audacity/assets/58490197/be666cd0-a534-460a-a25d-6a8ad57d9f6f)

**Reply by HighPriest** [reply]
So, I am diving deeper into the task at hand and apparently, nyquist plugins are unable to differentiate between individual tracks. So, to achieve desired effect, we might need to write a replacement script using ?VST? or some audacity built-in Cpp libraries.

I have no idea, but I'm looking.

**Comment by petersampsonaudacity** [comment]
See this outstanding bug:  

 **Label Sounds fails if the selection also includes a label track #5648**
